Sources/Sentry/SentryTracer.m

```diff
@@ -543,12 +578,16 @@ - (void)trimEndTimestamp
 
 - (SentryTransaction *)toTransaction
 {
+    // TODO: use middleware to create appStartSpans
```

Comment:
`m`: There is still a todo here.

Reply:
Yes. 
So, I was thinking about leaving this TODOs in the code as another way to track things that would be nice to have but are not related to the current PR, instead of opening issues.
Lets say we need to change appStartSpans behavior. By reading this TODO we could include this change in the same task. If we only have an open issue, we wont go back to the full board to see if there is something related to the same code being changed.

Me and @kahest talked about this, and he is onboard with the idea.

Reply:
We should use a specific format to tag these refactoring opportunities. Ideally we would refer to a GH issue like this
`// TODO(ref): use middleware to create appStartSpans https://github.com/getsentry/sentry-cocoa/issues/1234`

Reply:
Nice, I like the idea.

Reply:
I'm not a fan of having todos in the code. I prefer creating issues with references to the code.

Reply:
These issues are forgotten in limbo.